include/arch/arm/cortex_m/mpu/nxp_mpu.h

```diff
@@ -14,10 +14,12 @@
 #define NXP_MPU_REGION_NUMBER 12
 
 /* Read Attribute */
-#define MPU_REGION_READ  ((1 << 2) | (1 << 8) | (1 << 14))
+#define MPU_REGION_READ  ((1 << 2) | (1 << 8) | (1 << 14) | \
+			  (1 << 18) | (1 << 19) | (1 << 20))
```

Comment:
Execute isn't needed. Mike misunderstood my fix a little bit, my fault for not being completely clear. I suggested that he add bits 20,19,18 to the rwx macros, but I should have said respectively. In other words, add bit 20 to the read macro, bit 19 to the write macro, and bit 18 to the exec macro.

Regarding the magic numbers, there are additional helper macros in MK64F12.h for the sysmpu register fields. You can use SYSMPU_WORD_M3UM(x)